### ISSUE TITLE
Cant access site

### ISSUE THREAD

**Issue by tyler-dunkel** [almightycouch/audiopia#1]
your site doesn't serve across https.
Is this a future goal or done on purpose?

**Comment by almightycouch** [comment]
Hi there, currently, [beta.audiopia.io](http://beta.audiopia.io) is only a DNS `CNAME` record pointing to `origin.meteor.com`.
Meteor provides a very simple way to deploy applications to there own server using one single command. That what we're using right now.

When the project reaches a more productive state, we will deploy and host it on our own server and by the way provide SSL support.

**Comment by tyler-dunkel** [comment]
weird,
I thought that any app pushed to meteor.com would automatically be served
over https.
thanks

**Comment by ImaCrea** [comment]
Hi there, I'd love to try out your service but because HTTPS certificate lacking I can't access it. Are you planning to fix this or is there another I way I can try it please?

Thanks :)

**Comment by onny** [comment]
Yep site seems to be down. I dont even get a certificate warning :(
